specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2019-09-16/healthcare-apis.json

```diff
@@ -681,6 +681,24 @@
         "etag": {
           "description": "An etag associated with the resource, used for optimistic concurrency when editing it.",
           "type": "string"
+        },
+        "identity": {
+            "type": "object",
+            "description": "Setting indicating whether the service has a managed identity associated with it.",
+            "properties": {
+                "type":{
+                    "description" : "Type of identity being specified, currently SystemAssigned and None are allowed.",
+                    "type": "string",
+                    "enum": [
+                        "SystemAssigned",
+                        "None"
+                    ],
+                    "x-ms-enum": {
+                        "name": "ManagedServiceIdentityType",
+                        "modelAsString": false
```

Comment:
why modelAsString is false? it will cause SDK  breaking change if adding new enum value in future.

Reply:
Updated to model string true, thanks for the info.